fe1-web/ingestion/handlers/Default.ts

```diff
@@ -1,12 +1,11 @@
 import { ExtendedMessage } from 'model/network/method/message';
-import { ObjectType } from 'model/network/method/message/data';
-import { handleLaoMessage } from './Lao';
-import { handleWitnessMessage } from './Witness';
-import { handleMeetingMessage } from './Meeting';
-import { handleRollCallMessage } from './RollCall';
-import { handleElectionMessage } from './Election';
-import { handleChirpMessage } from './ChirpHandler';
-import { handleReactionMessage } from './ReactionHandler';
+import { MessageRegistry } from 'model/network/method/message/data';
+import { configure } from './index';
+
+// Create a registry and configure it
+const registry = new MessageRegistry();
+configure(registry);
```

Comment:
later, I believe we'll be able to just shift the configuration higher in 
https://github.com/dedis/popstellar/blob/master/fe1-web/ingestion/Configure.ts